include/IrrCompileConfig.h

```diff
@@ -411,6 +416,11 @@ B3D, MS3D or X meshes */
 #ifdef NO_IRR_COMPILE_WITH_PNG_WRITER_
 #undef _IRR_COMPILE_WITH_PNG_WRITER_
 #endif
+//! Define _IRR_COMPILE_WITH_TGA_WRITER_ if you want to write .tga files
+#define _IRR_COMPILE_WITH_TGA_WRITER_
+#ifdef NO_IRR_COMPILE_WITH_TGA_WRITER_

```

Comment:
The TGA writer is truly unused, don't add it back.

Reply:
Alright, removed it.